source/docs/essentials/extend-the-theme.md

```diff
@@ -1,6 +1,129 @@
 ---
+id: extend-the-theme
 title: Extend the theme
 ---
 
-> Content coming soon. See
-> [https://gitlab.com/front-commerce/front-commerce/issues/13](https://gitlab.com/front-commerce/front-commerce/issues/13)
+## Understanding theme overrides
+
+Front-Commerce web application is a fully featured e-commerce universal React application
+aimed at providing a sane base for your own theme.
+Even though you could reuse pages and components individually in a totally different theme,
+most of the time you might find easier and faster to start with the base theme and
+iterate from there.
+
+**Theme override is the mechanism allowing to extend, adapt, rewrite almost everything contained in the base theme.** It is what allows you to:
+- upgrade Front-Commerce and benefit from the latest component improvements,
+- customize how an extension is displayed,
+- or create a slightly different theme for the Black Friday in confidence!
+
+People having an understanding of [layout overrides in Magento](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-override.html), [child themes in Wordpress](https://developer.wordpress.org/themes/advanced-topics/child-themes/) / [Prestashop](https://devdocs.prestashop.com/1.7/themes/reference/template-inheritance/parent-child-feature/), [templates override in Drupal](https://www.drupal.org/docs/8/theming/twig/working-with-twig-templates), [themes in CakePHP](https://book.cakephp.org/3.0/en/views/themes.html) or other similar implementations should easily understand this mechanism in Front-Commerce.
+
+Your own theme will be located in its own folder and will use by default components from
+parent theme(s) — at least from Front-Commerce base theme.
+You would then copy the files you want to override in your theme folder by maintaining
+an identical file structure.
+Your component will be used instead of the base one anywhere across the application.
+
+Themes are part of a module’s web area. Hence you must follow theses steps:
+
+1. define the module in the base configuration file
+2. copy the file (`js`, `scss` or `gql`) you want to override in the `theme` folder of your module
+3. customize its content in your module directory
+
+## Define the module in the base configuration file
```

Comment:
I'd change this name by using something like:

```suggestion
## Configure your custom theme and the parent theme(s) used in your application
```

Because it would make more sense according to [L22](https://github.com/front-commerce/developers.front-commerce.com/pull/22/files#diff-a821e827b1dd1b2089cc6ac3dfc6067bR22).

If you accept this suggestion, the title in the above list should be changed too.

Reply:
Thanks!
I’ll go with « Configure your custom theme and use it in your application » for now.